keyboards/kbdfans/kbd6x/kbd6x.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2018 MechMerlin
+/* Copyright 2020 mixedfeelings
```

Comment:
```suggestion
/* Copyright 2018 MechMerlin
```

Reply:
It shouldn't be my name?

Reply:
No, because you have not significantly changed the file.

Reply:
ok! no worries, i only changed it on @mechmerlin 's suggestion

Reply:
oh i see! I am DUMMY! thanks for the catch